scripts/pelorus-operator-patches/07_spec_description.diff

```diff
@@ -80,6 +80,41 @@
 +              prometheus_storage_pvc_storageclass:
 +                description: Prometheus Persistent Volume storage class to be used.
 +                type: string
++              federated_prometheus_hosts:
++                description: 'Bla bla'
++                type: object
++                properties:
++                  instances:
++                    description: 'Bla bla'
++                    type: array
++                    items:
++                      type: object
++                      properties:
++                        id:
++                          description: 'Bla bla'
++                          type: string
++                        hostname:
++                          description: 'Bla bla'
++                          type: string
++                        password:
++                          description: 'Bla bla'
++                          type: string
++              external_prometheus_hosts:
++                description: 'Bla bla'
++                type: object
++                properties:
++                  instances:
++                    description: 'Bla bla'
++                    type: array
++                    items:
++                      type: object
++                      properties:
++                        id:
++                          description: 'Bla bla'
++                          type: string
++                        hostname:
++                          description: 'Bla bla'
++                          type: string
```

Comment:
I tested out adding this to Pelorus CRD (without the `+`s) and seems to work
![Screenshot from 2023-03-06 15-17-34](https://user-images.githubusercontent.com/66965232/223200529-11bf7041-6bcc-4fc3-80b9-8965d8ed2e22.png)